src/components/ImageView/index.native.js

```diff
@@ -43,7 +47,12 @@ class ImageView extends PureComponent {
 
     componentDidMount() {
         // Wait till animations are over to prevent stutter in navigation animation
-        InteractionManager.runAfterInteractions(() => this.calculateImageSize());
+        this.state.interactionPromise = InteractionManager.runAfterInteractions(() => this.calculateImageSize());
+    }
+
+    componentWillUnmount() {
+        if (!this.state.interactionPromise) { return; }
```

Comment:
I think our style guide says to not do this (ie: we should have return in a new line and the close bracket in another one.
Having said that, you can change this to: `this.state.interactionPromise && this.state.interactionPromise.cancel()`

Reply:
On that actually throws `Expected an assignment or function call and instead saw an expression.eslintno-unused-expressions` lint error.

Reply:
I changed it to return in new line.